src/main/java/blue/Blue.java

```diff
@@ -0,0 +1,41 @@
+package blue;
+
+import blue.task.TaskList;
+
+/**
+ * The main class for the Blue application, which manages tasks using a {@link TaskList} and a {@link Parser}.
+ * The application starts by greeting the user, then parsing user input to manage tasks, and finally saying farewell.
+ */
+public class Blue {
+    /** The task list managed by the application. */
+    private TaskList tasklist;
```

Comment:
I noticed that you named the variable tasklist here. According to naming conventions, it’s better to use taskList with camel case. Would it be clearer to rename this as taskList for consistency with standard Java conventions?